cmd/crc/cmd/config/config.go

```diff
@@ -46,6 +47,13 @@ func RegisterSettings(cfg *config.Config) {
 	}
 
 	disableEnableTrayAutostart := func(key string, value interface{}) string {
+		// If autostart-tray set to 'true' show message to run setup
+		if cast.ToBool(value) {
+			return fmt.Sprintf(
```

Comment:
If we can't make it work in the two ways, maybe we should simplify and always ask the user to do crc setup?

Reply:
Yeah, i think i'll go with this approach, will later try to implement the flow as described in https://github.com/code-ready/crc/pull/2064#issuecomment-791247341

Reply:
and for uninstalling/disable the user will have to do `crc cleanup` then `crc setup`